newIDE/app/src/InstancesEditor/ViewPosition.js

```diff
@@ -49,6 +49,16 @@ export default class ViewPosition {
     return this._height;
   }
 
+  containsPoint(x: number, y: number) {
+    const canvasPoint = this.toCanvasCoordinates(x, y);
+    return (
+      0 <= canvasPoint[0] &&
```

Comment:
just for the sake of not accessing the array twice, you could create variables `canvasPointX` and `canvasPointY` (easier to read too)

Reply:
I bet JS engines can do this micro optimization.
GDJS uses this way a lot. I agree that it take some time to get use to it. I'm not sure local variables will make it easier to read because they are more verbose in this situation.